<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Search</class>
 <widget class="QWidget" name="Widget_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_note">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background:rgb(219,230,244);
	border:0px;
	color:rgb(118,137,156);
	
	font: 10pt &quot;华文中宋&quot;;
}

QPushButton:focus{
	background:rgb(237,245,253);
	color:rgb(88,109,131);
}</string>
        </property>
        <property name="text">
         <string>文章</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_user">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background:rgb(219,230,244);
	color:rgb(118,137,156);
	border:0px;
	font: 10pt &quot;华文中宋&quot;;
}

QPushButton:focus{
	background:rgb(237,245,253);
	color:rgb(88,109,131);
}</string>
        </property>
        <property name="text">
         <string>用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_collection">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background:rgb(219,230,244);

	color:rgb(118,137,156);
	border:0px;
	font: 10pt &quot;华文中宋&quot;;
}

QPushButton:focus{
	background:rgb(237,245,253);
	color:rgb(88,109,131);
}</string>
        </property>
        <property name="text">
         <string>专题</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_notebook">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background:rgb(219,230,244);
	color:rgb(118,137,156);
	border:0px;
	font: 10pt &quot;华文中宋&quot;;
}

QPushButton:focus{
	background:rgb(237,245,253);
	color:rgb(88,109,131);
}</string>
        </property>
        <property name="text">
         <string>文集</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background:rgb(237,245,251);
border:white;
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>676</width>
        <height>406</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_prevPage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_nextPage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
